lib/no-code/code-generator.ts

```
@@ -0,0 +1,462 @@
+    }
+
+    return events;
+  }
+
+  private generateConstructor(): string[] {
+    const constructorNodes = this.nodes.filter(
+      (node) => node.label === "Constructor"
+    );
+    if (constructorNodes.length === 0) return [];
+
+    const constructor = constructorNodes[0];
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)